tests/test_download_firmware.py

```
@@ -3832,3 +3948,758 @@ def test_download_repo_prerelease_firmware_tracks_only_downloaded_when_failures(
+                return_value=[
+                    "firmware-2.7.23.7be5426",
+                    "firmware-2.7.23.4ee9598",
+                ],
+            ),
+            patch.object(
+                downloader,
+                "_download_prerelease_assets",
+                return_value=([success_result], [], False),
+            ),
+        ):
+            successes, failures, active_dir, summary = (
```

Ruff (0.15.12)

[warning] Unpacked variable `successes` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)